src/tests/ftest/control/dmg_pool_query_test.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+from dmg_utils import pool_query, get_pool_query_info
+from ior_test_base import IorTestBase
+from test_utils_pool import TestPool
+
+
+class DmgPoolQueryTest(IorTestBase):
+    """Test Class Description:
+    Simple test to verify the pool query command of dmg tool.
+    :avocado: recursive
+    """
+    # pylint: disable=too-many-ancestors
+    def setUp(self):
+        "Set up for dmg pool query."
+        super(DmgPoolQueryTest, self).setUp()
+
+        # Init the pool
+        self.pool = TestPool(self.context, dmg_command=self.get_dmg_command())
+        self.pool.get_params(self)
+        self.pool.create()
+
+        # Get the host list with port to provide the dmg command
+        self.port = self.params.get("port", "/run/server_config/*")
+        self.host_p = ["{}:{}".format(host, self.port)
+                       for host in self.hostlist_servers]
+
+    def test_pool_query_basic(self):
+        """
+        JIRA ID: DAOS-2976
+        Test Description: Test basic dmg functionality to query pool info on
+        the system. Provided a valid pool UUID, verify the output received from
+        pool query command.
+        :avocado: tags=all,tiny,pr,hw,dmg,pool_query,basic
```

Comment:
do you need a tag to specifically run this test?  i.e. poolquerybasic

Reply:
Ok, will add.